lib/oeqa/runtime/sanity/baseos.py

```diff
@@ -102,7 +103,35 @@ def test_baseos_systemd_process(self):
             status=1
         self.assertEqual(status, 0, msg="Error messages: %s" % output)
 
+    def test_baseos_systemd_boot_error(self):
+        ''' check systemd boot journal error''' 
+        known_issues_list = [
+            "GPT: Use GNU Parted to correct GPT errors",
+            # IOTOS-1575 [Edison] wpa_supplicant error during system booting
+            "Failed to open config file '/etc/wpa_supplicant/wpa_supplicant-wlan0.conf'",
+            "^\w{3,} \d{,2} \d{2}:\d{2}:\d{2} .* kernel: \w+",
```

Comment:
Using .* defeats the purpose because it matches also errors where the component printing the error is not "kernel", like:
```ffffff edison foobar: error in kernel: aaaa```

Please check again.

Also note that \w+ at the end is redundant.